ms/second-review-responses.tex

```diff
@@ -49,12 +49,37 @@
 
-\reply{Thank you for your comments. We have responded to each reviewer below. In response to reviewer 2 we have added several caveats to the GWAS section noting other sources of error such as allele frequency and LD variation among populations. Reviewer 3 raised two good points on our definition of neighborhood size and the interpetation of $\sigma$ -- first, that $\sigma$ refers to one- rather than two-dimensional distances between parent and offspring (this is consistent with previous theoretical work, and we have corrected our definition in the text), and second, that the use of separate draws for mate selection and dispersal means the total distance traveled by a locus along a pedigree link is greater for ``paternal'' than ``maternal'' material (taking ``maternal'' to mean the parent from which dispersal is calculated). Because our simulated genome is autosomal and individuals hermaphroditic and non-selfing, a locus will experience a mix of ``maternal'' and ``paternal'' transmission events along its genealogy. So we see this as a scaling issue rather than one of a ``bimodal'' dispersal kernal per se -- the expected axial displacement of a locus along a pedigree link is $\sqrt{3/2} \sigma$, rather than $\sigma$. However we note that in terms of summary statistics the effects of scaling $\sigma$ by $\sqrt{3/2}$ are weaker than those generated by factors like changing landscape resolution in lattice vs. continuous models (e.g. Appendix 1), so we don't think this significantly changes our interpretation of model outputs. }
+
+Thank you for your comments. We have responded to each reviewer below. In response to reviewer 2 we have added several caveats to the GWAS section noting other sources of error such as allele frequency and LD variation among populations. Reviewer 3 raised two good points on our definition of neighborhood size and the interpetation of $\sigma$ -- first, that $\sigma$ refers to one- rather than two-dimensional distances between parent and offspring (this is consistent with previous theoretical work, and we have corrected our definition in the text), and second, that the use of separate draws for mate selection and dispersal means the total distance traveled by a locus along a pedigree link is greater for ``paternal'' than ``maternal'' material (taking ``maternal'' to mean the parent from which dispersal is calculated). Because our simulated genome is autosomal and individuals hermaphroditic and non-selfing, a locus will experience a mix of ``maternal'' and ``paternal'' transmission events along its genealogy. So we see this as a scaling issue rather than one of a ``bimodal'' dispersal kernal per se -- the expected axial displacement of a locus along a pedigree link is $\sqrt{3/2} \sigma$, rather than $\sigma$.
+(Note that although the parent-offspring displacement is a mixture of mean-zero Gaussians
+with respect to a bimodal distribution of standard deviations,
+the norm of any such mixtures is never itself bimodal.)
```

Comment:
I think it's probably a good idea to say that it's not actually bimodal (and I finally put my finger on why it "feels" bimodal...).

Reply:
Also - we shoudl probably cut this paragraph down, because the quick summary ("R3 was right") is sufficient.

Reply:
i really like this last addition here saying specifically that there is no bimodality